etc/schema.json

```diff
@@ -6434,16 +6434,20 @@
                 },
                 "detect": {
                     "type": "object",
+                    "description": "Statistics related to the detection engines",
                     "additionalProperties": false,
                     "properties": {
                         "alert": {
-                            "type": "integer"
+                            "type": "integer",
+                            "description": "Count of alerts triggered"
                         },
                         "alert_queue_overflow": {
-                            "type": "integer"
+                            "type": "integer",
+                            "description": "Count of alerts discarded due to alert queue overflow"
                         },
                         "alerts_suppressed": {
-                            "type": "integer"
+                            "type": "integer",
+                            "description": "Count of alerts not logged due to noalert keyword usaged"
```

Comment:
Is this only for `noalert`? Or do suppressions/thresholds get counted here as well?

Reply:
Important catch, thanks.